presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -317,6 +317,7 @@ public class FeaturesConfig
     private boolean removeCrossJoinWithSingleConstantRow = true;
     private CreateView.Security defaultViewSecurityMode = DEFINER;
     private boolean useHistograms;
+    private HistoryBasedOptimizerInputStatisticsCheckStrategy historyBasedOptimizerInputStatisticsCheckStrategy = HistoryBasedOptimizerInputStatisticsCheckStrategy.ALWAYS;
```

Comment:
Should this configuration property be placed in `HistoryBasedOptimizationConfig`?